backend/src/packages/files/libs/helpers/create-invalid-file-error-message.helper.ts

```diff
@@ -0,0 +1,44 @@
+import { type MultipartFile } from '@fastify/multipart';
+
+import { type ValueOf } from '~/libs/types/types.js';
+
+import { FilesValidationError } from '../enums/enums.js';
+import { type FileInputConfig } from '../types/types.js';
+import { filesize } from './helpers.js';
+
+const createInvalidFileErrorMessage = (
+  errorType: ValueOf<typeof FilesValidationError>,
```

Comment:
Maybe better create separate class for such error than use helper?

Reply:
I suppose that error constructor should only take message and cause, but not fileInputConfig and the file itself, so helper is useful.